awscli/customizations/configure/__init__.py

```diff
@@ -36,3 +36,13 @@ def mask_value(current_value):
         return 'None'
     else:
         return ('*' * 16) + current_value[-4:]
+
+
+def profile_to_section(profile_name):
+    """Converts a profile name to a section header to be used in the config."""
+
+    if ' ' in profile_name:
+        section_format = 'profile "%s"'
```

Comment:
Given botocore uses `shlex.quote` we should probably use the corresponding `six.moves.shlex_quote` to be safe.